PSICover/PSICover.cs

```diff
@@ -160,21 +164,62 @@ class Analyzer {
                blocks.RemoveAt (i - 1);
          blocks.Reverse ();
 
+         int cHits = 0;

```

Comment:
We have to declare variable only before its use at valid point. In this case, this can be declared at line 170 before foreach block